database/insertData/_common/04_data_displays.js

```diff
@@ -76,7 +76,7 @@ exports.queries = [
               }
             ]
           }
-          outcomeDisplayTablesUsingId: {
+          dataDisplayTablesUsingId: {
```

Comment:
This could be a good timing to remove the inserts... Not sure if it is a good idea to keep this. We don't use this anymore...

Reply:
Yeah, it probably is, I was just a little nervous to get rid of them completely.